pmlc/dialect/pxa/ir/ops.cc

```diff
@@ -48,32 +48,56 @@ ParseResult parseKeywordIntoEnumAttr(OpAsmParser &parser,
 
 void AffineParallelOp::build(Builder *builder, OperationState &result,
                              ArrayRef<int64_t> ranges) {
-  SmallVector<AffineExpr, 8> lbExprs;
-  SmallVector<AffineExpr, 8> ubExprs;
-  // Make range expressions for each range
-  for (int64_t range : ranges) {
-    lbExprs.push_back(builder->getAffineConstantExpr(0));
-    ubExprs.push_back(builder->getAffineConstantExpr(range));
-  }
-  // Make the maps (handling the 0-dim case carefully)
+  // Default initalize empty maps
+  auto lb_map = AffineMap::get(builder->getContext());
+  auto ub_map = AffineMap::get(builder->getContext());
```

Comment:
nit: use `camelCase`

Reply:
Yup, trying to adjust my brain, will do.